82_Montauban/list.insee

```diff
@@ -1,8 +0,0 @@
-82121
```

Comment:
Is there a reason this one is deleted?

Reply:
Yes, the file `list.insee` is a duplicate of `insee.list` so I removed the file completely.